cg/store/filters/status_case_sample_filters.py

```diff
@@ -56,6 +57,25 @@ def get_not_sequenced_cases(case_samples: Query, **kwargs) -> Query:
     return case_samples.filter(Sample.last_sequenced_at == None).distinct()
 
 
+def get_sequenced_cases(case_samples: Query, **kwargs) -> Query:
+    not_sequenced = (
+        case_samples.filter(Sample.last_sequenced_at == None)
+        .with_entities(CaseSample.case_id)
+        .subquery()
+    )
+    return (
+        case_samples.outerjoin(not_sequenced, CaseSample.case_id == not_sequenced.c.case_id)
+        .filter(not_sequenced.c.case_id == None)
+        .distinct()
+    )
```

Comment:
Could you not just look for last_sequenced_at is not None?

Reply:
The query does rely on `last_sequenced_at`. The query is a bit tricky because a case is only defined as sequenced if all samples in it are sequenced. I think I'll keep it as is since it works and follows the structure of the other queries.

I have spent some time in the past trying to simplify these queries, but it is tricky 😅